modules/standard/Math.chpl

```diff
@@ -1057,6 +1057,66 @@ module Math {
     return truncf(x);
   }
 
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  extern proc j0(x: real(64)): real(64);
+
+  inline proc j0(x : real(32)): real(32) {
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
```

Comment:
Note that these doc-strings need to be pulled out of the procedure body in order to get rendered in the documentation. Right now it looks like this:

<img width="537" alt="screen shot 2018-01-10 at 11 00 20 am" src="https://user-images.githubusercontent.com/3595449/34782100-aa690e72-f5f5-11e7-9fa8-53ddfa85c46a.png">